source/Irrlicht/CIrrDeviceLinux.cpp

```diff
@@ -1485,7 +1485,7 @@ void CIrrDeviceLinux::createKeyMap()
 	KeyMap.push_back(SKeyMap(XK_Next, KEY_NEXT));
 	KeyMap.push_back(SKeyMap(XK_Page_Down, KEY_NEXT));
 	KeyMap.push_back(SKeyMap(XK_End, KEY_END));
-	KeyMap.push_back(SKeyMap(XK_Begin, KEY_HOME));
+	KeyMap.push_back(SKeyMap(XK_Begin, KEY_NUMPAD5));

```

Comment:
shouldn't this still be KEY_HOME? it's not the one on the numpad

Reply:
Good question, I just did it for symmetry without further thinking about it. How was the the old binding to `KEY_HOME` chosen?

It looks like `XK_Begin` only exists for symmetry, but I failed to find a use for it anywhere.

https://sourceforge.net/p/nebuladevice/mailman/message/5216679/
https://stackoverflow.com/questions/25259852/what-is-the-begin-key
https://bugs.java.com/bugdatabase/view_bug?bug_id=4850137 (the most helpful so far)

Maybe it is a solution to just remove the binding? It looks like there's no keyboard that actually has this key.

Reply:
Hmm, looks like you're right. This can stay as-is.